specifications/api/swagger.yaml

```diff
@@ -4109,6 +4186,7 @@ x-tagGroups:
   - name: Shelley (Shared Wallets)
     tags:
     - Shared Wallets
+    - Shared Keys
```

Comment:
Small note: I'd put the `Shelley (Shared Wallets)` section before `Byron (Random)`. Byron is legacy so it makes more sense to have it presented last :+1:

Reply:
done